src/comps/Navbar.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import '../styles/homepage/navbar.css'
+import { BsFilter } from 'react-icons/bs';
+import { FiSearch } from 'react-icons/fi';
+import { GrDocumentNotes } from 'react-icons/gr';
+import { IoMdMenu } from 'react-icons/io';
+
+function Navbar({ setHideAside }) {
+    
+  return (
+    <div className='Navbar'>
+        <div className="navbar_left">
+        <IoMdMenu size='1.5em' className='ham-menu_icon' onClick={() => setHideAside(prev => !prev)}/>
+        <GrDocumentNotes size='2em' className='notero-icon'/>
+        <h1 className="brandname">Notero</h1>
+        </div>
+        <div className="navbar_right">
+        <div className='searchbox'>
```

Comment:
Only suggestion is the placeholder text for the search bar has bad contrast when it comes to green bg.
Could be color white for the placeholder property only, I tried to find more about this here :
https://www.w3schools.com/howto/howto_css_placeholder.asp

Reply:
Good point, my bad i haven't noticed!